frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -4,7 +4,7 @@ import { Card } from "react-bootstrap";
 export default function CommonsOverview({ commons }) {
     return (
         <Card data-testid="CommonsOverview">
-            <Card.Header as="h5">Announcements</Card.Header>
+            <Card.Header as="h5"><strong>Announcements</strong></Card.Header>
```

Comment:
I think it might be preferable to separate styling from the markup. Perhaps you could consider removing the `<strong>` tag and using a CSS class instead?